### PR TITLE
changing prep to match the new curriculum

### PR DESCRIPTION
**Pull request by bundacia**
see [ch3684](https://app.clubhouse.io/learnersguild/story/3684/).

### CODE REVIEW

File: Runway/README.md [hunk 1]
```diff
@@ -94,6 +68,19 @@ Your focus should be on developing the above skills. The below tasks are good pl
...
 
+#### More FreeCodecamp!
+
+Check out these other useful [FreeCodeCamp](https://www.freecodecamp.com/) sections:
+
+- Responsive Design with Bootstrap (2-4 hours)
+- Basic Front End Development Projects (10-30 hours)
+- JSON APIs and Ajax (1-2 hours)
+- Intermediate Front End Development Projects (10-30 hours)
+ - Build a Random Quote Machine Incomplete
```
Comment (tannerwelsh) [inline Runway/README.md]: Let's get rid of these extra "Incomplete"s here

File: Runway/README.md [hunk 2]
```diff
@@ -1,58 +1,41 @@
...
 ## More Recommended Prep
 
-Done with FreeCodeCamp? Great. If you have the time and resources, double down on your prep by going through the following:
-
-#### Web Development Basics
-
-These fundamental skills are needed for nearly all web development projects:
+Done with all the required prep? Great. If you have the time and resources,
+keep going! These fundamental skills are needed for nearly all web development
+projects:
```
Comment (tannerwelsh) [inline Runway/README.md]: Consider changing the above line to more strongly encourage continual learning:

> Done with all the required prep? Great! But don't stop there! Continuous learning is a core part of being a developer. Keep practicing by building these fundamental web development skills:

File: Runway/README.md [hunk 3]
```diff
@@ -84,7 +59,6 @@ Your focus should be on developing the above skills. The below tasks are good pl
 #### Version Control and Project Management
 
 - Get oriented to GitHub with [Hello World][gh-hello-guide], [Understanding the GitHub Flow][gh-flow-guide]
```
Comment (tannerwelsh) [inline Runway/README.md]: I think these GitHub exercises should be required

File: Runway/README.md [hunk 4]
```diff
@@ -1,58 +1,41 @@
...
+- Complete the [Try Git][cs-try-git] course
 
-**Additional Prep Reading & Videos**
+#### Non-technical Prep
 
 - Read the Learner Guide section on [Cognitive Apprenticeship](https://guide.learnersguild.org/Game_Manual/Cognitive_Apprenticeship.html), including watching this [demonstration session](https://shereef.wistia.com/medias/dbknami46w)
-
-**Read the following sections of the [COS Guide](https://cos.learnersguild.org/)**:
+- Read the following sections of the [COS Guide](https://cos.learnersguild.org/):
  - [Introduction](https://cos.learnersguild.org/)
```
Comment (tannerwelsh) [inline Runway/README.md]: I think one more indent space is needed for this to register as a sub list of "Read the following sections..."

File: Runway/README.md [hunk 5]
```diff
@@ -1,58 +1,41 @@
...
+- HTML5 and CSS
+- Basic JavaScript
+- Object Oriented and Functional Programming
+- Basic Algorithm Scripting (try to get through about half)
 
+#### Development Environment and Tooling
+
+- Download and install the [Atom][atom] editor
+- Read chapters 1 and 2 of the [Atom Flight Manual][atom-flight-manual]
+- Go through Codecademy's [Learn the Command Line][codecademy-cli] course
```
Comment (tannerwelsh) [inline Runway/README.md]: Let's change this language to "Complete Codecademy's..." (just a bit more explicit about the expectations)

File: Runway/README.md [hunk 6]
```diff
@@ -1,58 +1,41 @@
...
-- Object Oriented and Functional Programming (1-2 hours)
-- Basic Algorithm Scripting (10-30 hours)
-- JSON APIs and Ajax (1-2 hours)
-- Intermediate Front End Development Projects (10-30 hours)
- - Build a Random Quote Machine Incomplete
- - Show the Local Weather Incomplete
+- HTML5 and CSS
+- Basic JavaScript
+- Object Oriented and Functional Programming
+- Basic Algorithm Scripting (try to get through about half)
```
Comment (tannerwelsh) [inline Runway/README.md]: Let's not say just half. I think all of these are good exercises for folks to have experience with. We can always pare it down more later if it's too much.

**Review by tannerwelsh — CHANGES_REQUESTED** [review]
Looking good, Trevor! I requested just a few additional modifications, otherwise I think this will be a good change. Might want to get @deadlyicon's eyes on this before a merge too.

**Comment by tannerwelsh** [comment]
One other thing, what about having folks complete some kind of small project? My first thought is that we could port the old enrollment game challenge (https://gist.github.com/lg-bot/f306837bb0fd958ef5de) into a repo and update the instructions so that folks fork the repo and work on the steps. This way they're actually getting some exposure to a real workflow rather than working in contained, in-browser environments for everything.

File: Runway/README.md (hunk 6 above)
Comment (bundacia) [reply Runway/README.md]: works for me

Comment (jaredatron) [reply Runway/README.md]: I'd remove `Object Oriented and Functional Programming` we don't really teach this anywhere explicitly anymore. I mostly recommend folks just do functional programming in JavaScript.

File: Runway/README.md (hunk 3 above)
Comment (jaredatron) [reply Runway/README.md]: yes! Some very basic Git and GitHub skills should be required.

File: Runway/README.md (hunk 1 above)
Comment (jaredatron) [reply Runway/README.md]: you need to indent two spaces here

File: Runway/README.md [hunk 7]
```diff
@@ -1,58 +1,41 @@
...
 
 - Read the Learner Guide section on [Cognitive Apprenticeship](https://guide.learnersguild.org/Game_Manual/Cognitive_Apprenticeship.html), including watching this [demonstration session](https://shereef.wistia.com/medias/dbknami46w)
-
-**Read the following sections of the [COS Guide](https://cos.learnersguild.org/)**:
+- Read the following sections of the [COS Guide](https://cos.learnersguild.org/):
  - [Introduction](https://cos.learnersguild.org/)
  - [Overview](https://cos.learnersguild.org/COS_Overview/) 
  - [Dynamic Tensions](https://cos.learnersguild.org/Dynamic_Tensions/) 
 
 ** Watch Karim Bishay's [presentation on Feedback](https://shereef.wistia.com/medias/8wx7n2fs65#)** (1 hour)
```
Comment (jaredatron) [inline Runway/README.md]: this line is not showing up as a bold line. I dont think you can have a space after the `**`

File: Runway/README.md (hunk 5 above)
Comment (bundacia) [reply Runway/README.md]: 👍

File: Runway/README.md (hunk 4 above)
Comment (bundacia) [reply Runway/README.md]: I think it's right how it is. This is how it renders for me:
readme_md

File: Runway/README.md (hunk 2 above)
Comment (bundacia) [reply Runway/README.md]: 👍

File: Runway/README.md (hunk 3 above)
Comment (bundacia) [reply Runway/README.md]: ok

**Comment by bundacia** [comment]
@tanner, that challenge is currently an in-browser activity as well, using js-bin, we could change that, but I'm hesitant to require too many things that are not in-browser. I don't want people to get stuck because they don't have access to a reliable computer they can install stuff on. I'd love for this prep to be able to be completed at a public library, for instance.

File: Runway/README.md (hunk 1 above)
Comment (bundacia) [reply Runway/README.md]: done

**Comment by tannerwelsh** [comment]
@bundacia yeah that's a good point. That was a criteria of mine too, now that you mention it. I think it's a worthwhile standard for us to hold ourselves to. Ok, so in that case I think this all looks good 👍

**Review by tannerwelsh — APPROVED** [review]

**Comment by tannerwelsh** [comment]
@deadlyicon assigning you to review since this should probs get the 👌 from you too

**Review by jaredatron — CHANGES_REQUESTED** [review]

File: Runway/README.md [hunk 8]
```diff
@@ -94,6 +66,19 @@ Your focus should be on developing the above skills. The below tasks are good pl
...
 
+#### More FreeCodecamp!
+
+Check out these other useful [FreeCodeCamp](https://www.freecodecamp.com/) sections:
+
+- Responsive Design with Bootstrap (2-4 hours)
+- Basic Front End Development Projects (10-30 hours)
+- JSON APIs and Ajax (1-2 hours)
+- Intermediate Front End Development Projects (10-30 hours)
+ - Build a Random Quote Machine
```
Comment (jaredatron) [inline Runway/README.md:76]: i think you need two spaces to indent here

**Review by jaredatron — CHANGES_REQUESTED** [review]

File: Runway/README.md (hunk 6 above)
Comment (tannerwelsh) [reply Runway/README.md]: @deadlyicon I think it's actually a good bit of prep, although the name is a little misleading. It just consists of exercises working with constructor functions, and then some basic "functional" array methods (sort, map, reduce). It's much more lowercase objects and functional programming than an introduction to the paradigms.

**Comment by tannerwelsh** [comment]
Thx @bundacia!

**Pull request merged**